src/MathOptFormat.jl

```diff
@@ -22,22 +26,30 @@ MOIU.@model(Model,
         MOI.PositiveSemidefiniteConeTriangle, MOI.PositiveSemidefiniteConeSquare,
         MOI.ExponentialCone, MOI.DualExponentialCone),
     (MOI.PowerCone, MOI.DualPowerCone, MOI.SOS1, MOI.SOS2),
-    (MOI.SingleVariable,),
+    (MOI.SingleVariable, Nonlinear),
     (MOI.ScalarAffineFunction, MOI.ScalarQuadraticFunction),
     (MOI.VectorOfVariables,),
     (MOI.VectorAffineFunction, MOI.VectorQuadraticFunction)
 )
 
+# TODO(odow): missing method in MOI/src/utilities/model.jl.
+MOI.supports(::InnerModel, ::MOI.ObjectiveFunctionType) = true
```

Comment:
@blegat is this just a missing method? Or is there some reason it's not implemented?

Reply:
It shouldn't be implemented because it is not copyable, see
https://github.com/JuliaOpt/MathOptInterface.jl/blob/38a4e883672641da575b284813254a61747c5854/src/attributes.jl#L800-L801
and
https://github.com/JuliaOpt/MathOptInterface.jl/blob/38a4e883672641da575b284813254a61747c5854/src/attributes.jl#L110-L112

Reply:
Weird. I swear it was failing tests without this before...

Reply:
@blegat , see the failing travis tests without this line.